### PR TITLE
Add on-enter handler

### PR DESCRIPTION
**Pull request by matklad**
Now, typing doc comments is much more pleasant

### CODE REVIEW

**Comment by matklad** [comment]
bors r+

Let's see how bad I am at appeasing tslint :D

File: editors/code/src/commands/index.ts [hunk 1]
```diff
@@ -13,5 +14,6 @@ export {
     matchingBrace,
     parentModule,
     runnables,
-    syntaxTree
+    syntaxTree,
+    on_enter,
```
Comment (aochagavia) [inline editors/code/src/commands/index.ts]: I guess *prettier* and *tslint* are going to complain... But if they don't, you should probably use `onEnter` and remove the `,`

**Comment by matklad** [comment]
bors try

Looks like bors/travis are having troubles with the recent introduction of an organization

**Comment by aochagavia** [comment]
@DJMcNab I don't think this PR should have passed the CI. The [logs](https://travis-ci.org/rust-analyzer/rust-analyzer/jobs/439162310) show a tslint error, though the build is green anyway.

**Comment by matklad** [comment]
bors try

**Pull request merged**